addons/project/models/project.py

```diff
@@ -1014,6 +1014,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
     legend_normal = fields.Char(related='stage_id.legend_normal', string='Kanban Ongoing Explanation', readonly=True, related_sudo=False)
     is_closed = fields.Boolean(related="stage_id.fold", string="Closing Stage", store=True, index=True, related_sudo=False, help="Folded in Kanban stages are closing stages.")
     parent_id = fields.Many2one('project.task', string='Parent Task', index=True)
+    ancestor_task_id = fields.Many2one('project.task', compute='_compute_ancestor_task_id', index=True, recursive=True, store=True)
```

Comment:
After having thought of it I would rather name it ancestor_id (as we are already in task model), what do you think?
Sorry for the change, I should have thought about it prior to give you the name 😕.

Reply:
Done.